lib/logging/activity-event.js

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const logger = require('./')('activity');
+
+module.exports = function activityEvent(event, uid, client_id, userAgent) {
+  // by the time this function is called, we'll always have a client_id somehow
+  var info = {
+    event: event,
+    uid: uid,
+    client_id: client_id,
+  };
+  // we're called from either a browser or a server, so check if we have this
+  if ( userAgent ) {
+    info.userAgent = userAgent;
+  }
+  logger.info('activityEvent', info);
```

Comment:
Not sure at the moment, am just looking. Will paste examples into the issue as a comparison.